@navikt/ds-css/modal/index.css

```diff
@@ -0,0 +1,56 @@
+.ReactModal__Body--open {
+  overflow: hidden;
+}
+.navds-modal {
+  --modal-border-radius: 4px;
+
+  background-color: var(--navds-color-white);
+  display: block;
+  padding: 1rem 1rem 1rem 1rem;
+  border-radius: var(--modal-border-radius);
+  position: relative;
+  flex-grow: 0;
+  overflow: auto;
+  max-height: 100%;
+  margin-bottom: 0;
+  z-index: var(--navds-z-index-modal-content);
+}
+.navds-modal:focus,
+.navds-modal--focus {
+  box-shadow: var(--navds-shadow-focus);
```

Comment:
Jeg trodde ikke selve modalen var et interaktivt element. Skal den ha ha focus-stil? Ref. det vi snakket om for popover.

Reply:
Slik modalen er nå idag så vil man sette fokus på hele modal-elementet når man klikker på det. Det er også default at man får fokus på hele modal når den blir åpnet. Tenker dette er noe som er laget for å tilfredstille skjermlesere. 

Når det kommer til hvordan vi setter fokus når `shouldCloseOnOverlayClick` er false er jeg enda ikke sikker på. Vi kan velge å ikke gjøre noe som helst med fokuset her, tror kanskje det er den beste løsningen for nå.